menu-bar/styles.css

```diff
@@ -0,0 +1,51 @@
+body {
+  margin: 0;
+  font-family: 'Source Sans Pro';
+  background-color: rgb(238, 238, 238);
+}
+
+header {
+  margin-left: 7.5%;
+  margin-right: 7.5%;
+  padding: 10px 100px;
+  background-color: rgb(31, 36, 50);
+}
+
+.right-side-nav {
+  float: right;
+}
+
+.nav-bar {
+  margin: 0;
+  padding-top: 6px;
+}
+
+.feature-and-security {
+  margin-right: 20px;
+  display: inline;
+  font-size: 90%;
+  color: rgb(187, 189, 193);
+}
+
+.feature-and-security-font-visual {
```

Comment:
You shouldn't need a separate class for the font/vizual styles. You can include them in `.feature-and-security`.

Reply:
I wrote it on a separate class because adding a link to the "Feature" and "Security" made the font look purple and added an underline that I did not want.

Reply:
You should be able to use CSS to address the issue.

Reply:
(Without extra classes)